### ISSUE TITLE
Hello, world?

### ISSUE THREAD

**Issue by virtualritz** [iomentum/cargo-breaking#1]
After installing:
```
❯ cargo install cargo-breaking
    Updating crates.io index
  Downloaded cargo-breaking v0.0.2
  Downloaded 1 crate (7.7 KB) in 1.69s
  Installing cargo-breaking v0.0.2
   Compiling cargo-breaking v0.0.2
    Finished release [optimized] target(s) in 27.59s
  Installing /Users/moritz/.cargo/bin/cargo-breaking
   Installed package `cargo-breaking v0.0.2` (executable `cargo-breaking`)
```
Inside a folder for a crate I published:
```
❯ cargo breaking
Hello, world!
```
I tried comparing branches, using `--against` but all I ever get is above `Hello, world!` message. What am I missing?

**Comment by o0Ignition0o** [comment]
Hey there!

It seems like we haven't published since we started (really) working on it, maybe @scrabsha can provide more input!

In the meantime i think you can clone the repo, and build it from the main branch.

There's a couple of things @scrabsha wanted to work on before the next publish, she probably has more details :)

**Comment by scrabsha** [comment]
Hi!

As o0ignition0o said, the changes you'll see in this repository are not published on crates.io, at least for now. If you want to try the lastest version, please clone the repo and checkout to [`scrabsha/iter-19-06-21`](https://github.com/iomentum/cargo-breaking/tree/scrabsha/iter-19-06-21). There is some initial work on the import resolver, but this is still very limited. It is also okay to use `main`, but you won't get the most recent improvements :).

Before actually releasing `cargo-breaking` on crates.io, I want to finish the import resolver as the whole project would be useless without this feature.

**Comment by scrabsha** [comment]
Oh, I guess I should also reflect this in the README

**Comment by scrabsha** [comment]
Fixed by e6b5ab5
